### ISSUE TITLE
Collision-aware In-hand 6D Object Pose Estimation using Multiple Vision-based Tactile Sensors. (arXiv:2301.13667v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#24570]
Collision-aware In-hand 6D Object Pose Estimation using Multiple Vision-based Tactile Sensors. (arXiv:2301.13667v1 [cs.RO])<br>
https://ift.tt/pvab4Nk<br>
<p>In this paper, we address the problem of estimating the in-hand 6D pose of an object in contact with multiple vision-based tactile sensors. We reason on the possible spatial configurations of the sensors along the object surface. Specifically, we filter contact hypotheses using geometric reasoning and a Convolutional Neural Network (CNN), trained on simulated object-agnostic images, to promote those that better comply with the actual tactile images from the sensors. We use the selected sensors configurations to optimize over the space of 6D poses using a Gradient Descent-based approach. We finally rank the obtained poses by penalizing those that are in collision with the sensors. We carry out experiments in simulation using the DIGIT vision-based sensor with several objects, from the standard YCB model set. The results demonstrate that our approach estimates object poses that are compatible with actual object-sensor contacts in $87.5\%$ of cases while reaching an average positional error in the order of $2$ centimeters. Our analysis also includes qualitative results of experiments with a real DIGIT sensor.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
